apps/microsoft-teams/contentful-app-manifest.json

```diff
@@ -1,5 +1,32 @@
 {
   "actions": [
+    {
+      "id": "handleAppEvent",
+      "name": "Handle incoming app events",
```

Comment:
Our other app actions start with `msteams` in the name (which honestly might be superfluous), but just pointing out that the names of the app actions feel a bit inconsistent.